src/plugins/intel_gpu/src/kernel_selector/cl_kernels/reorder_weights_int4.cl

```diff
@@ -5,7 +5,60 @@
 #include "include/batch_headers/fetch_weights.cl"
 
 KERNEL(reorder_weights_int4)(const __global INPUT0_TYPE* input, __global OUTPUT_TYPE* output) {
-#if defined(INPUT0_LAYOUT_IOYX) && defined(OUTPUT_LAYOUT_OIYX)
+#if defined(INPUT0_LAYOUT_OIYX) && defined(OUTPUT_LAYOUT_OIYX)
+
+#if OUTPUT_X_PITCH != 1
+#define OUTPUT_INNERMOST_PITCH OUTPUT_X_PITCH
```

Comment:
I guess proper name is not `INNERMOST_PITCH` but `SECOND_INNERMOST_PITCH`. Maybe it is getting too long..

Reply:
Change the name. please review it again.